src/main/java/knightminer/simplytea/data/gen/WorldgenGenerator.java

```diff
@@ -0,0 +1,23 @@
+package knightminer.simplytea.data.gen;
+
+import knightminer.simplytea.core.Registration;
+import net.minecraft.core.HolderLookup;
+import net.minecraft.core.RegistrySetBuilder;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.data.PackOutput;
+import net.minecraftforge.common.data.DatapackBuiltinEntriesProvider;
+import net.minecraftforge.registries.ForgeRegistries;
+
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+public class WorldgenGenerator extends DatapackBuiltinEntriesProvider {
+    public static final RegistrySetBuilder BUILDER = new RegistrySetBuilder()
+            .add(Registries.CONFIGURED_FEATURE, Registration::configuredFeatures)
+            .add(Registries.PLACED_FEATURE, Registration::placedFeatures)
+            .add(ForgeRegistries.Keys.BIOME_MODIFIERS, Registration::biomeModifiers);
```

Comment:
Any reason these need to be defined in registration over just methods in this class? I like to keep code only used in datagen in separate classes when reasonable.

Reply:
No reason, didn't know if you wanted them in the same class or not